src/dashboard/src/components/access/views.py

```diff
@@ -94,10 +94,17 @@ def all_records(client, request, system=''):
     search_pattern = request.GET.get('title', '')
     identifier = request.GET.get('identifier', '')
 
-    return helpers.json_response(client.find_collections(search_pattern=search_pattern,
-                                                         identifier=identifier,
-                                                         page=page,
-                                                         page_size=page_size))
+    records = client.find_collections(
+        search_pattern=search_pattern,
+        identifier=identifier,
+        page=page,
+        page_size=page_size,
+    )
+
+    for r in records:
+        r = _get_sip_arrange_children(r, system)
```

Comment:
I think this worked by accident because `_get_sip_arrange_children` mutates the passed record, but I'll update it to reassign the contents of `record` for clarity.